### ISSUE TITLE
How can we pass url as parameter using typescript. While trying so I am getting an error. 

### ISSUE THREAD

**Issue by soumya-gobant** [DevExpress/testcafe#4042]
Please find the code snippet:
import { Selector } from 'testcafe'
import config from '../tests/config.json'

//selectors
const name = Selector('#developer-name');
const submitButton = Selector('#submit-button');

fixture `Getting Started`
    .page `${config.baseUrl}`;

test('My first test', async t => {
    await t
    .typeText(name, 'John Smith')
    .click(submitButton);
});

Smaple json:
{
    "baseUrl": "http://devexpress.github.io/testcafe"   
}

but I am not facing same issue with js. Not able to resolve above issue. it is throwing me error "Expected 1 arguments, but got 2".

**Comment by AlexKamaev** [comment]
I was able to reproduce the issue. It looks like an issue in TypeScript, not in TestCafe. I can reproduce it even without TestCafe. I created a sample code:
```JS
const config = { baseUrl: "http://devexpress.github.io/testcafe" };

function log(arg1) {
}

log `${config.baseUrl}`;

```
Then I tried to compile this code using the `tsc` util and I got the equal message:
`error TS2554: Expected 1 arguments, but got 2.`

I recommend you rewrite the `.page` definition in the following manner:
```JS
fixture `Getting Started`
    .page(config.baseUrl);
```
I should resolve the issue.
